pkg/tests/rabbitmq/rabbitmq_direct_test.go

```diff
@@ -0,0 +1,151 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ormushq/ormus/pkg/broker/messagebroker"
+	"github.com/ormushq/ormus/pkg/broker/rabbitmq"
+)
+
+// Define a TestCase struct to hold parameters for each test case
+type DirectTestCase struct {
+	Name         string
+	ExchangeName string
+	Kind         string
+	NumWorkers   int
+	NumMessages  int
+	ExpectedMsg  int
+}
+
+func TestRabbitMQConcurrentConsumption(t *testing.T) {
+	// Define test cases
+	directTestCases := []DirectTestCase{
+		{
+			Name:         "SingleWorker",
+			ExchangeName: "test_exchange",
+			Kind:         "direct",
+			NumWorkers:   1,
+			NumMessages:  10,
+			ExpectedMsg:  10,
+		},
+		{
+			Name:         "MultipleWorkers",
+			ExchangeName: "test_exchange",
+			Kind:         "direct",
+			NumWorkers:   10,
+			NumMessages:  100,
+			ExpectedMsg:  1000,
+		},
+	}
+	fmt.Println("start tests...")
+	// Run test cases
+	for _, tc := range directTestCases {
+		t.Run(tc.Name, func(t *testing.T) {
+			runTest(t, tc)
+			time.Sleep(10 * time.Second)
+		})
+	}
+	fmt.Println("All tests completed successfully")
```

Comment:
Please remove Prints after debugging this makes CI test page messy and out of hand.